flight/Modules/System/systemmod.c

```diff
@@ -67,28 +65,61 @@
 #if defined(PIOS_SYSTEM_STACK_SIZE)
 #define STACK_SIZE_BYTES PIOS_SYSTEM_STACK_SIZE
 #else
-#define STACK_SIZE_BYTES 924
+#define STACK_SIZE_BYTES 1024
 #endif
 
-#define TASK_PRIORITY PIOS_THREAD_PRIO_LOW
+#define TASK_PRIORITY PIOS_THREAD_PRIO_NORMAL
```

Comment:
Before, the event system used to run at high priority to pump periodic callbacks and to pump events.  SystemMod ran at low priority and did very little-- configuration checks and blinked an LED.

This PR changes things somewhat.

* There's no task switch involved in pumping a callback, so it runs at whatever priority the task was that generated the callback.  However it holds an important lock (the UAVO lock), so it is likely to get promoted to the priority of any pre-empting threads by priority inheritance.  IMO this is just about perfect.
* Periodic callbacks that used to run at high priority now run at normal priority in the system task.  It doesn't look like that is bad for any of the existing periodic callbacks.
* The configuration check and heartbeat LED run at normal priority.  IMO bumping the configuration check frequency is good, but one "feature" is lost.  If highest/high/normal tasks are using all the CPU, the "heartbeat" LED will stop or slow down in current code.  Bumping it to priority normal means that this will not happen as much if normal prio tasks are crowded.

There's one thing that is kinda bad with current code and this PR.  If a low priority task pushes something to a queue that goes to a high priority task, a bad little choreographed sequence of events happens.  The low priority task gets preempted, there's a (expensive) task switch to the high priority task, which attempts to get the uavo lock, blocks, increases the priority of the low priority task, which completes its actions and unlocks, and then (expensively) switches to the expensive task again.  This is bad for both throughput and latency.  Figuring out how to cure this is too tricky for now, though.